CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## [UNRELEASED]
+### Added
+- Added a changelog
+
+### Changed
+- create orders without a contract_id
+- add get operation for orders
```

Comment:
I am a fan of sorting, because i think it makes it a lot easier to understand what happened. So i would prefer to keep the categories. I also would opt for the meaning of **changed** and **added** that KeepAChangelog uses. It's a new feature, so i would understand that as added and not a change of behaviour, because it did not exist before.